pkg/kf/options.yml

```diff
@@ -14,11 +14,8 @@ configs:
     type: string
     description: skip the detect buildpack step and use the given name
   - name: DockerImage
```

Comment:
I'd prefer to use a new variable rather than re-use an existing one if we're going to change its purpose and type.